src/foraging_gui/ValidateCheckpointSaving.py

```diff
@@ -0,0 +1,192 @@
+import json
+import os
+import numpy as np
+
+def load_json(file):
+    """Load a json file and return the data as a dictionary"""
+    with open(file, 'r') as f:
+        data = json.load(f)
+    return data
+
+def load_old_style(testfile):
+    """Load the old style file and return the data as a dictionary"""
```

Comment:
I assume "old style" means "entire json saved at the end of the session. Can you use a more descriptive name than "old style"?